embedded/devices/include/mppt.h

```diff
@@ -0,0 +1,81 @@
+#include <linux/can.h>
+#include <pthread.h>
+
+#include "can.h"
+
+class mppt {
```

Comment:
Sorry, this is picky but let's make it Mppt with a capital M. This would need to be updated everywhere else as well

Reply:
Please let's kill the undefined MPPT acronym. Either in the README documentation, a comment above the class, or even better, both. We should be doing this for all acronyms in our code base